### PR TITLE
Adding boilerplate code

### PR DESCRIPTION
**Pull request by shub-garg**
Adding standard Django start up code.

### CODE REVIEW

File: ServiceFinder/ServiceFinder/__pycache__/__init__.cpython-312.pyc
Comment (ahhcash) [inline ServiceFinder/ServiceFinder/__pycache__/__init__.cpython-312.pyc:1]: we should probably add this entire directory to .gitignore

File: ServiceFinder/db.sqlite3
Comment (ahhcash) [inline ServiceFinder/db.sqlite3:1]: guessing this was added via the boilerplate as well. this would be a good for our dev environment

**Review by ahhcash — CHANGES_REQUESTED** [review]
Minor changes. Looks good overall

File: ServiceFinder/ServiceFinder/__pycache__/__init__.cpython-312.pyc
Comment (shub-garg) [reply ServiceFinder/ServiceFinder/__pycache__/__init__.cpython-312.pyc:1]: Done

File: ServiceFinder/db.sqlite3
Comment (shub-garg) [reply ServiceFinder/db.sqlite3:1]: Yes!

File: ServiceFinder/ServiceFinder/settings.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-xjni&&k5_)8zqujz6y(kqw!8rajdyz4_y*ru*ml3=6qo_ht!uv'
```
Comment (ahhcash) [inline ServiceFinder/ServiceFinder/settings.py:23]: should this be here? 😳

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+ServiceFinder/ServiceFinder/__pycache__/
```
Comment (ahhcash) [inline .gitignore:1]: how does the directory structure work? Do we need the outer ServiceFinder directory?

File: ServiceFinder/ServiceFinder/settings.py (hunk 1 above)
Comment (KartikeySharma) [reply ServiceFinder/ServiceFinder/settings.py:23]: Found this resource for putting `SECRET_KEY `in `env`: [Link](https://stackoverflow.com/questions/15209978/where-to-store-secret-keys-django)

**Review by KartikeySharma — CHANGES_REQUESTED** [review]
Apart from @aakashshankar's comment, LGTM

**Comment by ahhcash** [comment]
let's close this PR for now, reopen this later

**Pull request closed without merging**